scripts/gap9-run.sh

```
@@ -0,0 +1,543 @@
+
+	log_success "Host-side USB/IP setup complete"
+}
+
+cmd_start_usbip_host() {
+	cmd_setup_usbip_host
+
+	log_info "Starting host-side USB/IP server (pyusbip)..."
+	log_info "This process will run in the foreground. Press Ctrl+C to stop."
+
+	cd "$PYUSBIP_DIR" &&
+		# shellcheck disable=SC1091
```

Shellcheck (0.11.0)

[error] Couldn't parse this simple command. Fix to allow more checks.

(SC1073)